monitoring/cluster_observability_operator/installing-the-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-cluster-observability-operators"]
+= Installing the Cluster Observability Operator
+include::_attributes/common-attributes.adoc[]
+:context: installing_the_cluster_observability_operator
+
+toc::[]
+
+:FeatureName: Using the Cluster Observability Operator
```

Comment:
This wording feels awkward to me given how the technology preview snippet is written. I don't see "using" something itself as a feature. Is it inaccurate to state that the Cluster Observability Operator itself is a tech preview feature?

Reply:
Thanks @kowen-rh. I agree. I'll update the wording so that the Operator itself is stated as the feature